src/lesson15/hw/Main.java

```diff
@@ -0,0 +1,55 @@
+package lesson15.hw;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+ * В главном классе создать 5 объектов класса Car
+ * Создать ArrayList и использовать методы: add(), addAll(), set(), indexOf(), size(),
+ * contains(), get(), remove(), lastIndexOf(), isEmpty(), clear() для этих объектов.
+ * */
+public class Main {
+    public static void main(String[] args) {
+
+        // Create an objects of Car type
+        Car car1 = new Car("Mitsubishi", 15, "Red", new Owner("Bruce", 45, 25));
+        Car car2 = new Car("Porche", 1, "Black", new Owner("Chuc", 35, 17));
+        Car car3 = new Car("Volga", 35, "Grey", new Owner("Valera", 56, 44));
+        Car car4 = new Car("Pagani", 7, "Blue", new Owner("Alberto", 70, 50));
+        Car car5 = new Car("Tesla", 10, "Green", new Owner("Elon", 52, 30));
+
+        List<Car> carList = new ArrayList<>(); //create an ArrayList of Car type objects
+        // add a Car object
+        carList.add(car1);
```

Comment:
it's shorrter:  carList.add(new Car("Mitsubishi", 15, "Red", new Owner("Bruce", 45, 25)))